pwchem/protocols/General/protocol_converter.py

```diff
@@ -183,4 +184,57 @@ def _summary(self):
                 summary.append('Converted to Cif')
             elif self.outputFormatTarget.get() == 2:
                 summary.append('Converted to Mol2')
-        return summary
\ No newline at end of file
+        return summary
+    
+    
+class ConvertMDSystem(EMProtocol):
+    """
+    Convert an MD simulation to a specific file format using VMD
+    """
+
+
+    _label = 'Convert MD system format'
+    _program = ""
+
+    def _defineParams(self, form):
+        form.addSection(label='Input')
+
+        form.addParam('inputSystem', PointerParam, pointerClass="MDSystem",
+                      label='MD System:', allowsNull=False,
+                      help="Any system with a system file and trajectory file that VMD can read")
+
+        form.addParam('outputFormat', EnumParam, default=0,
+                       choices=['DCD'],
+                       label='Output format',
+                       help = "The only allowed output format is currently dcd")
+
+    # --------------------------- Steps functions --------------------
+
+    def _insertAllSteps(self):
+        self._insertFunctionStep('convertStep')
+
+    def convertStep(self):
+        inSystem = self.inputSystem.get()
+        
+        fnStructure = inSystem.getSystemFile()
+        fnStructRoot = os.path.splitext(os.path.split(fnStructure)[1])[0]
+        fnStructOut = self._getPath(fnStructRoot + '.pdb')
+        
+        traj_empty = mdtraj.load(fnStructure, top=fnStructure) 
+        traj_empty.save_pdb(fnStructOut)
+
+        outSystem = MDSystem(filename=fnStructOut)
+
+        if inSystem.hasTrajectory():
+            fnTrj = inSystem.getTrajectoryFile()
+            
+            traj = mdtraj.load(fnTrj, top=fnStructOut)
+            
+            fnTrjRoot = os.path.splitext(os.path.split(fnTrj)[1])[0]
+            fnTrjOut = self._getPath(fnTrjRoot + '.dcd')
+            traj.save_dcd(fnTrjOut)
+            
+            outSystem.setTrajectoryFile(fnTrjOut)
```

Comment:
The resulting trajectory system has pdb and dcd files, with no topology file, is that correct? Does the viewer of MDSystem work for this?

Reply:
oh, I didn't check that. It could be possible to use the pdb file as a topology file too maybe. That's how I'm using it in mdtraj. I'll have a look

Reply:
There doesn't seem to be a general md system viewer

Reply:
There doesn't seem to be any ability to parse gromacs top files by vmd, mdtraj or mdanalysis. I could possibly write one for mdtraj based on the psf one.

Reply:
or maybe for prody although then you'd have a dependency there. I was also thinking about putting the converter in scipion-em-prody so maybe that could be a way to go

Reply:
If you want to add it there too, I'm okay. But I do think it is useful in chem, it will be the default analysis tool. At some point I will add OpenMM and seems like they work fine together

Reply:
ok. Sounds good

Reply:
oh, it looks like OpenMM can handle gromacs files!
https://parmed.github.io/ParmEd/html/omm_gromacs.html

Reply:
Parmed seems like a very general library for trajectory files that could be a good addition to scipion-chem, maybe instead of mdtraj

Reply:
oh, Parmed doesn't know the trajectory formats so we still need both